.travis.yml

```diff
@@ -29,11 +35,19 @@ script:
   # Build maestro
   - ./build.sh
 
+  # Run devicedb in docker
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      docker run --rm -d -p 127.0.0.1:9090:9090/tcp senramakri/test_images:devicedb_nocluster
+      sleep 10
+    fi
+
   # Run unit tests
   - cd "../../${upstream}"
   - DEBUG=1 DEBUG2=1 ./build.sh
   - ( cd sysstats && LD_LIBRARY_PATH=../../maestro/vendor/github.com/armPelionEdge/greasego/deps/lib/ go test -v )
   - ( cd maestroConfig && LD_LIBRARY_PATH=../../maestro/vendor/github.com/armPelionEdge/greasego/deps/lib/ go test -v )
+  - ( cd vendor/github.com/armPelionEdge/maestroSpecs && go test -v )
```

Comment:
Making sure, does this not need the LD_LIBRARY_PATH env var set prior to running?

Reply:
Nope, that doesnt need the path ad it doesnt depend on the lib.